docs/home.md

```diff
@@ -13,6 +13,9 @@ and [Security Assertion Markup Language(SAML) 2.0](http://docs.oasis-open.org/se
 
 ?> While OAuth 2.0 SSO setup is available on the [TechPass portal](https://portal.techpass.gov.sg), SAML service is available only by request. If you need SAML support for your product and services, submit a [service request](https://go.gov.sg/techpass-sr). We will get back to you within three business days.
 
+## TechPass is not...
+TechPass is not an email service provider. TechPass is a purely an Identity provider to allow users to access the downstream SGTS services
+
```

Comment:
Can this be added as a note.

> **Note**:
>- TechPass is not an email service provider. It is purely an Identity provider that authenticates users accessing the SGTS services.

Reply:
hmmm.. lol.. can.. I'm just wondering if there are other misunderstanding of techpass.. so thought to create a section for it.. hahhah..

Reply:
i think i will just added it as an additional sentence to the intro. cos there's already a note above.. the flow feels weird if i include it in there